### ISSUE TITLE
support for H1 heading

### ISSUE THREAD

**Issue by edublancas** [ploomber/soorgeon#29]
Some notebooks use H1 headings as separators, but soorgeon looks for H2. Maybe have an heuristic an determine which one to use

**Comment by idomic** [comment]
@edublancas What happens if you have both?

**Comment by edublancas** [comment]
good point. maybe we should have an option and let the user decide? I cannot think of a good heuristic
